elastic4s-tests/src/test/scala/com/sksamuel/elastic4s/search/queries/ScriptScoreQueryTest.scala

```diff
@@ -0,0 +1,57 @@
+package com.sksamuel.elastic4s.search.queries
+
+import com.sksamuel.elastic4s.requests.script.Script
+import com.sksamuel.elastic4s.testkit.DockerTests
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.util.Try
+
+class ScriptScoreQueryTest extends AnyWordSpec with DockerTests with Matchers {
+
+  Try {
+    client.execute {
+      deleteIndex("person")
+    }.await
+  }
+
+  client.execute(
+    bulk(
+      indexInto("person") fields(
+        "name" -> "reese",
+        "age" -> 1.0
+      ),
+      indexInto("person") fields(
+        "name" -> "finch",
+        "age" -> 1.0
+      ),
+      indexInto("person") fields(
+        "name" -> "finch",
+        "age" -> 2.0
+      ),
+      indexInto("person") fields(
+        "name" -> "finch",
+        "age" -> 3.0
+      )
+    ).refreshImmediately
+  ).await
+
+  "script score query" should {
+    "filter by query" in {
+      client.execute {
+        search("person") postFilter {
+          scriptScoreQuery(termQuery("name", "finch"))
+            .script(Script("doc['age'].value % 3"))
+        }
+      }.await.result.totalHits shouldBe 3
+    }
+    "sort by score" in {
```

Comment:
This test is failing currently. The score appears to be unaffected.

Reply:
Found my error. I'd copy/pasted `search(<indexname>) postFilter ...` rather than `search(<indexname>) query ...`